sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EncodingExtensions.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+
+namespace Azure.Messaging.EventHubs.Primitives
```

Comment:
If you'd be so kind, doc comments would be appreciated on the class and all methods.